modules/oe_webtools_maps/src/Plugin/Field/FieldFormatter/WebtoolsMapFormatter.php

```diff
@@ -71,11 +129,13 @@ public function settingsSummary() {
    */
   public function viewElements(FieldItemListInterface $items, $langcode) {
     $element = [];
+    $config = $this->configFactory->get('oe_webtools_maps.settings');
 
     foreach ($items as $delta => $item) {
       $data_array = [
         'service' => 'map',
-        'version' => '2.0',
+        // Fallback to version 2.0 for BC.
+        'version' => $config->get('map_version') ?? '2.0',
         'map' => [
           'zoom' => $this->getSetting('zoom_level'),
           'center' => [
```

Comment:
```
if ($this->getSetting('show_marker')) {
        $data_array['layers'] = [
```
should be updated according to changes in Webtools Maps UEC for maps 3.0. See this changes: https://github.com/openeuropa/oe_webtools/pull/258/files#diff-d5788593f82f730c0aae0cc4840492e4a812488fa1fb087015bec48283327268L90

Reply:
I confirm that, without this change, the marker does not appear on the map.